Simplified/NYPLBookRegistry.m

```diff
@@ -662,7 +662,12 @@ - (NSArray *)heldBooks
 
 - (NSArray *)myBooks
 {
-  return [self booksMatchingStateMask:~(NYPLBookStateHolding | NYPLBookStateUnregistered)];
+  return [self booksMatchingStateMask:
+          (NYPLBookStateDownloadNeeded
+           | NYPLBookStateDownloading
+           | NYPLBookStateDownloadFailed
+           | NYPLBookStateDownloadSuccessful
+           | NYPLBookStateUsed)];
 }
```

Comment:
So if I understand this correctly, accessing `myBooks` or `heldBooks` should reliably give me what I expect in each of those tabs in the UI, WITHOUT having to perform any further array filtering? When I was looking at the code inside the view controllers for those two classes, I think at the time I was less convinced that different code maintainers were interpreting things differently..